api/views/CloudRecordSummaryView.py

```diff
@@ -86,6 +90,18 @@ def get(self, request, format=None):
          end_date,
          global_user_name) = self._parse_query_parameters(request)
 
+        # Check that at most one of group_name, service_name
+        # and global_user_name is set
+        parameters_to_check = (group_name, service_name, global_user_name)
+
+        # for each variable in parameters_to_check that
+        # is not none, increment set_count
```

Comment:
Bit of a trivial comment as you're describing the code and also inaccurate, as you're not incrementing but summing a list and you're not looking for `not None` just `None`. This is one reason why you should avoid describing code in comments.